packages/shared/apis/Permissions.ts

```diff
@@ -26,14 +26,21 @@ export class Permissions extends ExposableAPI {
     //  Only the two permissions that start with ALLOW_TO_... can be conceed without user
     //  interaction
     const json = this.parcelIdentity.land.sceneJsonData
-    const list = json.requiredPermissions || []
-    if (list.indexOf(test) !== -1 &&
+    const list = (json.requiredPermissions || []) as PermissionItem[]
+    if (list.includes(test) &&
       (test === PermissionItem.ALLOW_TO_MOVE_PLAYER_INSIDE_SCENE ||
         test === PermissionItem.ALLOW_TO_TRIGGER_AVATAR_EMOTE)) {
       return true
     }
 
-    return this.permissionGranted.indexOf(test) !== -1
+    return this.permissionGranted.includes(test)
+  }
+  /**
+   * Returns if it has many permissions
+   */
+  @exposeMethod
+  async hasManyPermissions(test: PermissionItem[]): Promise<boolean> {
+    return test.filter(item => this.hasPermission(item)).length == test.length
```

Comment:
this could be replaced with a `every`. And also rename the param to what it is 😛 
```ts
return permissionItems.every(item => this.hasPermission(item))
```

Reply:
I replaced it with explicit for, it's more readable (because the promises, await, etc)